### DISCUSSION TITLE
i send  invited email  , but  why is not my server url  ....

### DISCUSSION THREAD

**Discussion by au-top** [supabase/supabase#5756] [Questions] [answered]
i check link url , he is not link to my server url , why .....
😭

<img width="1081" alt="image" src="https://user-images.githubusercontent.com/44134842/156325667-731d4ace-2dc4-4b70-8848-868634ec6f78.png">

**Comment by GaryAustin1** [comment, marked as answer]
I assume you are running your own supabase install. 
I've not messed around much with the local install, but seems some install variable is set wrong.  
To get you something to look at:
https://github.com/supabase/supabase/discussions/4381
